docker/lite/Dockerfile.ubi7.mysql80

```diff
@@ -0,0 +1,88 @@
+ && rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona.1 ${GNUPGHOME}/RPM-GPG-KEY-Percona.2 ${GNUPGHOME}/RPM-GPG-KEY-CentOS-7 ${GNUPGHOME}/RPM-GPG-KEY-MySQL /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release \
+ && curl -L --retry-delay 10 --retry 3 -o /tmp/mysqlrepo.rpm https://dev.mysql.com/get/mysql80-community-release-el7-3.noarch.rpm \
+ && curl -L --retry-delay 10 --retry 3 -o /tmp/perconarepo.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
+ && rpmkeys --checksig /tmp/mysqlrepo.rpm /tmp/perconarepo.rpm \
+ && rpm -Uvh /tmp/mysqlrepo.rpm /tmp/perconarepo.rpm \
+ && rm -f /tmp/mysqlrepo.rpm /tmp/perconarepo.rpm
+RUN curl -L --retry-delay 10 --retry 3 -o /tmp/libev.rpm http://mirror.centos.org/centos/7/extras/x86_64/Packages/libev-4.15-7.el7.x86_64.rpm \
+ && curl -L --retry-delay 10 --retry 3 -o /tmp/gperf.rpm http://mirror.centos.org/centos/7/os/x86_64/Packages/gperftools-libs-2.6.1-1.el7.x86_64.rpm \
+ && curl -L --retry-delay 10 --retry 3 -o /tmp/numactl.rpm http://mirror.centos.org/centos/7/updates/x86_64/Packages/numactl-libs-2.0.12-3.el7_7.1.x86_64.rpm \
+ && curl -L --retry-delay 10 --retry 3 -o /tmp/sysstat.rpm http://mirror.centos.org/centos/7/updates/x86_64/Packages/sysstat-10.1.5-18.el7_7.1.x86_64.rpm \
+ && curl -L --retry-delay 10 --retry 3 -o /tmp/strace.rpm http://mirror.centos.org/centos/7/os/x86_64/Packages/strace-4.12-9.el7.x86_64.rpm \
+ && rpm -i --nodeps /tmp/libev.rpm /tmp/gperf.rpm /tmp/numactl.rpm /tmp/sysstat.rpm /tmp/strace.rpm \
+ && rm -f /tmp/libev.rpm /tmp/gperf.rpm /tmp/numactl.rpm /tmp/sysstat.rpm /tmp/strace.rpm
+RUN yum update \
+ && yum install --setopt=alwaysprompt=no --setopt=tsflags=nodocs bzip2 ca-certificates gnupg libaio libcurl \
+    jemalloc gperftools-libs procps-ng rsync wget openssl hostname curl tzdata make \
+# Can't use alwaysprompt=no here, since we need to pick up deps
+#   No way to separate key imports and accept deps separately in yum/dnf
+ && yum install -y --setopt=tsflags=nodocs --enablerepo mysql80-community --disablerepo mysql57-community \
+    mysql-community-client mysql-community-server \
+# Have to use hacks to ignore conflicts on /etc/my.cnf install
+ && mkdir -p /tmp/1 \
+ && yum install --setopt=alwaysprompt=no --downloadonly --downloaddir=/tmp/1 --enablerepo mysql80-community --disablerepo mysql57-community percona-xtrabackup-80 \
+ && rpm -Uvh --replacefiles /tmp/1/*rpm \
+ && rm -rf /tmp/1 \
+ && yum clean all \
+ && yum clean all --enablerepo mysql80-community --disablerepo mysql57-community \
+ && rm -rf /etc/my.cnf /var/lib/mysql /tmp/gpg /sbin/mysqld-debug
+
+# Set up Vitess user and directory tree.
+RUN groupadd -g 1001 -r vitess && useradd -r -u 1001 -g vitess vitess
+RUN mkdir -p /vt/vtdataroot && chown -R vitess:vitess /vt
+
+# Set up Vitess environment (just enough to run pre-built Go binaries)
+ENV VTROOT /vt/src/vitess.io/vitess
+ENV VTDATAROOT /vt/vtdataroot
+ENV PATH $VTROOT/bin:$PATH
+ENV MYSQL_FLAVOR MySQL80
```

Comment:
It should be possible to remove `MYSQL_FLAVOR`. Vitess has been able to autodetect it for a while, but some of the legacy python tests still required it being set.